feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/autodiscovery/item/ContentItems.kt

```diff
@@ -52,6 +52,8 @@ internal fun LazyListScope.contentItems(
                 AccountOAuthView(
                     onOAuthResult = { result -> onEvent(Event.OnOAuthResult(result)) },
                     viewModel = oAuthViewModel,
+                    isEnabled = (state.autoDiscoverySettings?.isTrusted ?: false) ||
+                        state.configurationApproved.value ?: false,
                 )
```

Comment:
Maybe extract the components of the expression to variables to make it more readable?

```suggestion
                val areSettingsTrusted = state.autoDiscoverySettings?.isTrusted == true
                val areSettingsApproved = state.configurationApproved.value ?: false

                AccountOAuthView(
                    onOAuthResult = { result -> onEvent(Event.OnOAuthResult(result)) },
                    viewModel = oAuthViewModel,
                    isEnabled = areSettingsTrusted || areSettingsApproved,
                )
```

(I have no preference between `== true` and `?: false`. Only presented as an option to consider.)